x-pack/plugins/ml/public/application/components/help_menu/index.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
+
+export const HelpMenu = React.memo(() => {
+  const { chrome, docLinks } = useKibana().services;
+
+  useEffect(() => {
+    chrome.setHelpExtension({
+      appName: i18n.translate('xpack.ml.chrome.help.appName', {
+        defaultMessage: 'Machine learning',
+      }),
+      links: [
+        {
+          content: i18n.translate('xpack.ml.chrome.helpMenu.documentation', {
+            defaultMessage: 'Machine learning documentation',
```

Comment:
I think it would fit better if we shorten this:
```suggestion
            defaultMessage: 'Documentation',
```